app/services/aozora-conflicts.js

```diff
@@ -0,0 +1,20 @@
+import Service, { inject as service } from '@ember/service';
+import { get } from '@ember/object';
+
+export default Service.extend({
+  ajax: service(),
+
+  list() {
+    return get(this, 'ajax').request('/users/_conflicts', {
+      method: 'GET'
+    });
+  },
+
+  resolve(chosen) {
+    return get(this, 'ajax').request('/users/_conflicts', {
+      method: 'POST',
+      dataType: 'json',
+      data: { chosen }
+    });
+  }
+});
```

Comment:
This could probably be a util over a service, doesn't hold any state.

Reply:
The `conflicts` state could be reused across the redirect & the conflicts list so that slide loads instantly?

Reply:
Hmm, not sure what you mean -- are you asking if it would? Services in ember are singletons.

Reply:
I'm thinking that's a benefit of having it be a service, that it would already be loaded once it changes to the conflicts list

Reply:
But you're not caching the result anywhere, so both times it would re-execute the request.

Another way might be to just pass the conflict list to the component and go the util route.

Reply:
Hmm, that's also a reasonable option.  How would I get access to the AJAX object from within a util?

Reply:
`import request from 'ember-ajax/request';`

Reply:
But the ajax service is overridden to add headers (which we need)

Reply:
Ah right, mmm I guess if the only place we're using it is the sign-in component, maybe we just move the code there?

Reply:
That's already a really big component, it seems like the last thing we'd want to do is muddy it with yet more logic